src/components/FilterAndSort.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import {useData} from '../contexts/DataContext'
+import '../styles/filterandsort.css'
+const FilterAndSort = () => {
+    const {state : {sortBy , showInventory , fastDelivery , isRated , isPriced}, dispatch} = useData()
+    return (
+        <div className = 'filter__and__sort'>
+            <div className = 'clear-filters' onClick = {() => dispatch({type : 'CLEAR_FILTER'})}>
+                CLEAR FILTERS
```

Comment:
make it more visible! or increase font-weight!

Reply:
Ohhk